src/scripts/towns/NPC.ts

```diff
@@ -1,16 +1,23 @@
 type NPCOptionalArgument = {
     requirement?: Requirement | MultiRequirement | OneFromManyRequirement,
     image?: string,
+    saveKey?: string,
 };
 
 class NPC {
     public talkedTo = ko.observable<boolean>(false); // Used for custom quests
```

Comment:
Am I blind or is this no longer updated by anything

Reply:
You right. Forgot to re-add it after toying with something. Fixed.